evennia/contrib/mapbuilder.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+
+"""
+
+Evennia World Builder
+
+Contribution - Cloud_Keeper 2016
+
+This is a command capable of taking a reference to a basic 2D ASCII
+map stored in the Evennia directory and  generating the rooms and exits 
+necessary to populate that world. The characters of the map are iterated
+over and compared to a list of trigger characters. When a match is found
+it triggers the corresponding instructions. Use by importing and including 
+the command in your default_cmdsets module. For example:
+
+    # mygame/commands/default_cmdsets.py
+    
+    from evennia.contrib import mapbuilder
+    
+    ...
+    
+    self.add(mapbuilder.CmdMapBuilder())
+
+You then call the command in-game using the path to the module and the
+name of the variable holding the map string. The path you provide is
+relative to the evennia or your mygame folder.
+
+    @mapbuilder <path.to.module.VARNAME>
+
+For example to generate from the sample map in this module:
+
+    @mapbuilder evennia.contrib.mapbuilder.EXAMPLE_MAP
+
+Whilst this map is contained here for convenience, it is suggested
+that your map be stored in a separate stand alone module in the
+mygame/world folder accessed with @mapbuilder world.gamemap.MAP
+
+The rooms generated for each square of the map are generated using 
+instructions for each room type. These instructions are intended 
+to be changed and adapted for your purposes. In writing these 
+instructions you have access to the full API just like Batchcode.
+
+"""
+# ---------- #
+
+# This code should be in mymap.py in mygame/world.
+
+# -*- coding: utf-8 -*-
+
+EXAMPLE_MAP = """\
+≈≈≈≈≈≈≈≈≈
+≈≈≈≈≈≈≈≈≈
+≈≈♣♠♣♠♣≈≈
+≈≈♠n∩n♠≈≈
+≈≈♣∩▲∩♣≈≈
+≈≈♠n≈n♠≈≈
+≈≈♣♠≈♠♣≈≈
+≈≈≈≈≈≈≈≈≈
+≈≈≈≈≈≈≈≈≈
```

Comment:
There seems to be no instructions in `build_map` for how to build `≈` or am I just missing it? If it's using the default, this might be good to mention.